src/cli.js

```diff
@@ -58,16 +59,15 @@ const connect = (options) => {
 
     if (!isTaken) {
       const child = fork(
-        require.resolve('./startServer.js'), args, {/* silent: true */}
+        require.resolve('./startServer.js'), args, { silent: true }
```

Comment:
This was the reason things weren't working at first. Everything seems to work with this turned on. Is this fine?